plugins/network-elements/nuage-vsp/src/com/cloud/agent/api/sync/SyncDomainAnswer.java

```diff
@@ -17,28 +17,38 @@
 // under the License.
 //
 
-package com.cloud.agent.api.guru;
-
-import java.util.List;
-import java.util.Map;
+package com.cloud.agent.api.sync;
 
 import com.cloud.agent.api.Answer;
-import com.cloud.agent.api.Command;
 
-public class ReserveVmInterfaceVspAnswer extends Answer {
+public class SyncDomainAnswer extends Answer {
+
+    private final boolean _success;
 
-    public List<Map<String, String>> _interfaceDetails;
+    public SyncDomainAnswer(boolean success) {
+        super();
+        this._success = success;
+    }
 
-    public ReserveVmInterfaceVspAnswer(Command cmd, List<Map<String, String>> interfaceDetails, String details) {
-        super(cmd, true, details);
-        this._interfaceDetails = interfaceDetails;
+    public boolean getSuccess() {
+        return _success;
     }
 
-    public ReserveVmInterfaceVspAnswer(Command cmd, Exception e) {
-        super(cmd, e);
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        SyncDomainAnswer that = (SyncDomainAnswer) o;
+
+        if (_success != that._success) return false;
+
+        return true;
     }
 
-    public List<Map<String, String>> getInterfaceDetails() {
-        return this._interfaceDetails;
+    @Override
+    public int hashCode() {
```

Comment:
This ``hashCode`` implementation should include attributes from the ``Command`` super class.